sui/src/bin/wallet.rs

```diff
@@ -80,7 +80,7 @@ async fn try_main() -> Result<(), anyhow::Error> {
             "Config file [{:?}] doesn't exist, do you want to connect to a Sui Gateway [yn]?",
```

Comment:
```suggestion
            "Config file [{:?}] doesn't exist, do you want to connect to a Sui Gateway [yN]?",
```
This shows the default better, because if I enter "zzz" I get a "no".